packages/asset/test/AssetReveal.test.ts

```diff
@@ -66,8 +66,8 @@ describe('AssetReveal (/packages/asset/contracts/AssetReveal.sol)', function ()
         newMetadataHashes1,
         [revealHashA]
       );
-      expect(result.events[3].event).to.equal('AssetRevealMint');
-      const newTokenId = result.events[3].args.newTokenIds[0];
+      expect(result.events[4].event).to.equal('AssetRevealMint');
+      const newTokenId = result.events[4].args.newTokenIds[0];
```

Comment:
I thought we changed the AssetReveal tests to remove finding events this way? Ideally we would finding the events by name (not events index - this is brittle if we make further changes to the contracts) - @wojciech-turek